crates/turborepo-lib/src/commands/generate.rs

```diff
@@ -0,0 +1,60 @@
+use std::process::{Command, Stdio};
+
+use anyhow::Result;
+
+use crate::{child::spawn_child, cli::GenerateCommand};
+
+fn verify_requirements() -> Result<()> {
+    let output = Command::new("npx")
+        .arg("--version")
+        .stdout(Stdio::null())
+        .stderr(Stdio::null())
+        .status();
+
+    const ERROR_MSG: &str = "Unable to run generate - missing requirements (npx)";
+    match output {
+        Ok(result) => {
+            if result.success() {
+                return Ok(());
+            }
+            return Err(anyhow::anyhow!(ERROR_MSG));
+        }
+        Err(_) => {
+            return Err(anyhow::anyhow!(ERROR_MSG));
+        }
+    }
+}
```

Comment:
```suggestion
    match output {
        Ok(result) if result.success() => Ok(()),
        _ => {
            Err(anyhow::anyhow!("Unable to run generate - missing requirements (npx)"))
        }
    }
```

Reply:
Any reason this didn't get applied?

Reply:
Oh what....maybe I staged it and didn't apply, thanks for double checking!